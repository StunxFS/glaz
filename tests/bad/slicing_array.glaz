fn main() {
	_ = vec[i32]{ 1, 2, 3, 4 }[:];
}
