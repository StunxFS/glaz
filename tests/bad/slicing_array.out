tests/bad/slicing_array.glaz:2:7: error: invalid slice expression for array
     | 
   2 | 	_ = vec[i32]{ 1, 2, 3, 4 }[:];
     | 	    ^^^^^^^^^^^^^^^^^^^^^^^^^
     = help: consider calling the `clone` method instead
glaz error: could not compile `slicing_array`, aborting due to previous error.
